### DISCUSSION TITLE
Fail to init pods

### DISCUSSION THREAD

**Discussion by ykecheng** [airflow-helm/charts#638] [Questions & Answers] [answered]
Hello,

I'm trying to build airflow with KubernetesExecutor on my cluster.
I'm using defult [sample-values-KubernetesExecutor.yaml](https://github.com/airflow-helm/charts/blob/main/charts/airflow/sample-values-KubernetesExecutor.yaml)

Also, I would to use  my company's Mysql as externalDatabase.

I can use helm to install/upgrade, but I always got following error:
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/108354781/186854900-8e634a86-d212-411a-b013-551dd856038e.png">

I've get into node and ping mysql server without any issue, looks like pods even not initilized

Below is my yaml file seting:

[custom-values.txt](https://github.com/airflow-helm/charts/files/9431099/custom-values.txt)

https://gist.github.com/ykecheng/e82147b562c9534e1d0f5c460a35fe4e

Thank you in advance!

**Comment by thesuperzapper** [comment, marked as answer]
@ykecheng I really need more information non to help.

Perhaps you can start by providing:
- `kubectl describe pod ...` for the failing pods (especially so we can see their events)
- `kubectl logs` (possibly also using `-p` to see the logs from the last crash loop)

**Reply by ykecheng** [reply]
Thanks! I've resoloved it.

**Reply by thesuperzapper** [reply]
@ykecheng do you mind sharing what was wrong, so you can help others?

**Reply by Adaverse** [reply]
I'm also getting the same issue.

**Reply by thesuperzapper** [reply]
@ykecheng I would love if you could post what your issue was to help others?

**Reply by ykecheng** [reply]
@thesuperzapper 
Actully it's my company Mysql connection issue. 
I got the correct privilege then it was resolved.
Thank you!
